aiohttp/web_urldispatcher.py

```diff
@@ -163,6 +163,9 @@ def __init__(self, name, prefix, directory, *,
             raise ValueError(
                 "No directory exists at '{}'".format(self._directory))
 
+        if os.environ.get("AIOHTTP_NOSENDFILE") == "1":
```

Comment:
I'm concerned that definition of this environment variable switch is differs from other environment variable switches used in asyncio/aiohttp.

`AIOHTTP_NO_EXTENSIONS` is [checked](
https://github.com/KeepSafe/aiohttp/blob/230bdf30d84fdda03f8285704d09f58bbdc984e5/aiohttp/multidict.py#L367) in the following way:

```
if bool(os.environ.get('AIOHTTP_NO_EXTENSIONS')):
```

At least inside `aiohttp` library environment variables switches should work in the same way.